database/mongoseed.sh

```diff
@@ -3,6 +3,9 @@
 seed_db="${MONGO_DB:-dev}"
 echo Dropping DB $seed_db
 mongo "$seed_db" --eval "db.dropDatabase()"
+mongo "$seed_db" --eval "db.getCollection('products').createIndex({category: 1, store: 1})"
```

Comment:
Do we have any evidence that the seeding actually makes any difference? I'd hope it does, but I'm curious.